library/weno/src/TransmitData.php

```diff
@@ -0,0 +1,97 @@
+<?php
+/**
+ * TransmitData class.
+ *
+ * @package OpenEMR
+ * @link    http://www.open-emr.org
+ * @author  Sherwin Gaddis <sherwingaddis@gmail.com>
+ * @copyright Copyright (c) 2016-2017 Sherwin Gaddis <sherwingaddis@gmail.com>
+ * @license https://github.com/openemr/openemr/blob/master/LICENSE GNU General Public License 3
+ */
+
+
+namespace OpenEMR\Rx\Weno;
+
+class TransmitData
+{
+    
+    public function __construct()
+    {}
+    
+    public function getDrugList($pid, $date)
+    {
+
+        $sql = "SELECT * FROM prescriptions WHERE patient_id = ? AND ntx = 1 AND txDate = ?";
+        $res = sqlStatement($sql, array($pid, $date));
+        return $res;
+    }
+
+    public function checkList($send)
```

Comment:
This function doesn't appear to be used anywhere and is odd (to start with a sqlFetchArray). If it's not used, then would remove it.

Reply:
Removed